### DISCUSSION TITLE
[RRFC] Can we discuss changes coming to `npm link` here?

### DISCUSSION THREAD

**Discussion by richardeschloss** [npm/rfcs#143] [General]
I think there are some good ideas discussed here: https://github.com/npm/rfcs/blob/latest/accepted/0011-npm-link-changes.md

I find the current behavior of `npm link` to be useful to me for local development, especially for testing my package before I publish; i.e., I get to to test out what the end user will feel when installing my package. It's also less writing for me than `ln -s target path`, and just works when I have to change my platform.

`npm install --link` is incredibly useful when it works. It can be made more useful if it saved the things it just installed to my package.json. 

In short, I don't think a breaking change needs to be introduced. I think terminology and language of the commands are important.
